### ISSUE TITLE
Training on different structures

### ISSUE THREAD

**Issue by simonjuleseric2** [imlab-uiip/lung-segmentation-2d#4]
Thank you for sharing this. Is there anyway to go about if you want to train the model to recognize several structures (e.g. lungs and heart)?

**Comment by eclique** [comment]
Sure. One way to do it is to produce two masks instead of one. This can be done by changing number of filters to 2 here: https://github.com/imlab-uiip/lung-segmentation-2d/blob/85a2ef53198b73d1e2d415479bb9764ec9496fe4/build_model.py#L51
You would also need to change the activation from sigmoid to softmax and loss from binary cross-entropy to categorical cross-entropy.
